docs/README.md

```diff
@@ -67,17 +65,16 @@ own.
 	The `docker/docker` repository contains a `Dockerfile` and a `Makefile`.
 	Together, these create a development environment in which you can build and
 	run a container running the Docker documentation website. To build the
-	documentation site, enter `make docs` at the root of your `docker/docker`
-	fork:
-	
+	documentation site, enter `make docs` in the `docs` directory of your `docker/docker` fork:
```

Comment:
You have to be in docs to `make docs` --- some projects added this to root Makefile not all of them. So yes, this is the universal way to do it.